engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/inputSettings/InputSettingsScreen.java

```diff
@@ -94,6 +86,35 @@ public void initialise() {
         mainLayout.setVerticalSpacing(8);
         mainLayout.setFamily("option-grid");
 
+        UIButton azerty = new UIButton();
+        azerty.setText(translationSystem.translate("${engine:menu#input-settings-apply}"));
+        azerty.subscribe(event -> {
+            BindCommands.AZERTY.forEach(this::setPrimaryBind);
+            bindsManager.registerBinds();
+        });
+        UIButton dvorak = new UIButton();
+        dvorak.setText(translationSystem.translate("${engine:menu#input-settings-apply}"));
+        dvorak.subscribe(event -> {
+            BindCommands.DVORAK.forEach(this::setPrimaryBind);
+            bindsManager.registerBinds();
+        });
+        UIButton neo = new UIButton();
+        neo.setText(translationSystem.translate("${engine:menu#input-settings-apply}"));
+        neo.subscribe(event -> {
+            BindCommands.NEO.forEach(this::setPrimaryBind);
+            bindsManager.registerBinds();
+        });
+        mainLayout.addWidget(new UILabel("specialKeyboards", "subheading", translationSystem.translate("${engine:menu#special-keyboards}")));
```

Comment:
The preferred way to edit UI files is by manually altering the `.ui` files for the screen you are after. [In your case that would be this one](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/resources/assets/ui/menu/inputSettingsScreen.ui)
If you need help with how to use our UI system then [check out our tutorial](https://github.com/Terasology/TutorialNui) or [hop onto our discord](http://discord.gg/Terasology)